prow/apis/prowjobs/v1/types.go

```diff
@@ -53,7 +53,7 @@ type ProwJobState string
 const (
 	// TriggeredState means the job has been created but not yet scheduled.
 	TriggeredState ProwJobState = "triggered"
-	// PendingState means the job is scheduled but not yet running.
+	// PendingState means the job is currently running and we are waiting for it to finish.
```

Comment:
I've dreamt about fixing this at some point. We should have a `Running` state, right?

Reply:
Yeah so very technically it may not be running but e.G. `pending` if the cluster has no capacity or sth. But I find this super confusing, because `triggered` to me sounds like it was what `pending` is (We `triggered` the test workload and are waiting for it).